public/app/features/plugins/components/AppRootPage.tsx

```diff
@@ -34,7 +34,7 @@ interface Props {
   // The ID of the plugin we would like to load and display
   pluginId: string;
   // The root navModelItem for the plugin (root = lives directly under 'home')
-  pluginNavSection: NavModelItem;
+  pluginNavSection?: NavModelItem;
```

Comment:
Can do but this was basically not required before. Like the code works already without `pluginNavSection` and I didn't have to do much changes, just change the typing so I wonder if there is some other cases where this would be undefined already.

Reply:
Hmm, that's a good point. I'll try to to think about it which other scenarios could cause this to be undefined!